plugin/trino-firebolt/pom.xml

```diff
@@ -0,0 +1,189 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.trino</groupId>
+        <artifactId>trino-root</artifactId>
+        <version>419</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>trino-firebolt</artifactId>
+    <description>Trino - Firebolt</description>
+    <packaging>trino-plugin</packaging>
+
+    <properties>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+    </properties>
+
+    <repositories>
+        <repository>
+            <id>repsy</id>
+            <name>Firebolt Private Maven Repository on Repsy</name>
```

Comment:
I dont think we allow repository adds to a specific plugin. Can you get your artifacts published to Maven Central please?

Reply:
I'm looking into publishing on Maven Central, so far I think we should be able to do so without issues. Will update the PR once it's done.

Reply:
thank you

Reply:
Removed this as we've migrated to Maven Central.